.github/workflows/build_centos7.yml

```diff
@@ -63,4 +66,4 @@ jobs:
           . venv/bin/activate
           export LC_ALL=en_US.UTF-8
           export LANG=en_US.UTF-8
-          make pythontest
\ No newline at end of file
+          make pythontest
```

Comment:
This extra newline character at the end of the file is unrelated to the rest of the PR